File: IdPhoto.js
const UniqueItem = require("./UniqueItem");

class IdPhoto extends UniqueItem {
  constructor(name) {
    super(name);
  }

  displayInfo() {
    console.log(`Informations de la photo d'identité :`);
    console.log(`Nom : ${this.name}`);
  }
}

module.exports = IdPhoto;

File: src/main/java/org/example/model/IdPhoto.java
package org.example.model;

public class IdPhoto extends UniqueItem {

    public IdPhoto(String newName) {
        super(newName);
    }

    @Override
    public String toString() {
        return "IdPhoto{" +
                "name='" + getName() + '\'' +
                '}';
    }
}
